### DISCUSSION TITLE
Standup (Tuesday, April 26, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#24] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Finish #14
 - Write down full dressed use cases

**What are your priorities for the next 24 hours?**:
 - Finish #21 

**Blockers**:
 - We didn't define a design system

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #23 with @BrandonMagana (pending review)

**What are your priorities for the next 24 hours?**:
 - Finish #17 

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Made progres on the aspirant search #22 .
 - Also adjusted some of the use cases for the aspirant seaarch.

**What are your priorities for the next 6 hours?**:
 - Finished #22 and the intro??? Document???

**Blockers**:
 - Redefine problem

**Shoutouts**:
 - @paotg123 for the nice design

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - I completed our three activity diagrams #15 
 - Started signup mockup with @CarlosCRG19 🏔
 - 
**Blockers**:
 - Can't deal with Figma UI 😡

**Shoutouts**:
 - @DavBot02, @CristianCazares , @paotg123 and @BrandonMagana  for being so quick with their mockups.

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Finished #23 with @DavBot02
 - Finished #13, it is still pending to be reviewed

**What are your priorities for the next 24 hours?**:
 -  Finished #17 

**Blockers**:
 - None 

**Shoutouts**:
 - @DavBot02  for helping me finished the job offer mock up

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 -  Started #22

**What are your priorities for the next 24 hours?**:
 - finish #22
